### ISSUE TITLE
CUSA04943 - GRAVITY RUSH™ 2

### ISSUE THREAD

**Issue by C4ndyF1sh** [shadps4-emu/shadps4-game-compatibility#1453]
### Game Name

GRAVITY RUSH™ 2

### Game code

CUSA04943

### Game version

01.00

### Used emulator's version (only released versions are acceptable)

0.5.0

### Current status

Ingame

### Operating System

MacOs

### Error

/

### Description

boots up and goes ingame but crashes 2 seconds later

### Screenshots

<img width="1710" alt="Screenshot 2024-12-26 at 00 58 26" src="https://github.com/user-attachments/assets/bac2b0fd-8629-45f6-a56b-6e8b1ac6093e" />

### Log File

[shad_log.txt](https://github.com/user-attachments/files/18247516/shad_log.txt)

**Comment by jarrett-g** [comment]
Should be closed - duplicate of #76

**Comment by C4ndyF1sh** [comment]
@jarrett-g Wrong, it's not a duplicate. This is on Mac and multiple reports about the same game are allowed if they have a different Serial number or have been tested on a different os

**Comment by Hermiten** [comment]
I prefer Menu label for this one
